web/src/components/StyledToolbarItem.tsx

```diff
@@ -3,12 +3,12 @@ import styled from 'styled-components'
 import dimensions from '../constants/dimensions'
 import CleanAnchor from './CleanAnchor'
 
-const StyledToolbarItem = styled(CleanAnchor)`
+const StyledToolbarItem = styled(CleanAnchor)<{ disabled?: boolean }>`
```

Comment:
- Instead of disabling the button I can just disable the cursor from the styling but this will affect the Tooltip from showing...

I noticed the `$` but I removed it just to write less redundant code like `$disabled:boolean` and `disabled:boolean` one for styling and the other for the button itself ..

Reply:
Maybe passing null for href and onclick is enough to disable it without using disabled for buttons

Reply:
> * Instead of disabling the button I can just disable the cursor from the styling but this will affect the Tooltip from showing...
No, we should definitely also disable the functionality of the button because there will always be users who'll use it anyway and then be disappointed that it doesn't work properly, plus it's more accessible.

> Maybe passing null for href and onclick is enough to disable it without using disabled for buttons
I don't quite understand, you mean change it to show up as `disabled` if the props for `href` or `onClick` are `null`? I guess that would work but I think it's easier to understand if we keep the `disabled` prop. What do you think?

Reply:
> I don't quite understand, you mean change it to show up as `disabled` if the props for `href` or `onClick` are `null`? I guess that would work but I think it's easier to understand if we keep the `disabled` prop. What do you think?

I mean I will keep the `disabled` prop and I will change the props for `StyledToolbarItem` to 
`href={isDisabled ? null : href}`
`onClick={isDisabled ? null : onClick}`
If there is no href or onClick whether it's a button or an anchor the user won't be able to navigate anywhere.

Reply:
But we also want to make it clear to the user that they can't click on the disabled item, so it shouldn't just be that you click on it and nothing happens.